tests/test_wave_03.py

```diff
@@ -54,13 +58,10 @@ def test_friends_unique_movies_not_duplicated():
 
     # Assert
     assert len(friends_unique_movies) == 3
+    assert friends_unique_movies == [FANTASY_4, HORROR_1, INTRIGUE_3]
```

Comment:
Be careful using `==` for asserts on lists. If, for some reason, the movies end up in a different order, this test would fail even if all of the elements are the same. `==` requires that the ordering of the elements be the same as well. Instead, consider statements like `assert FANTASY_4 in friends_unique_movies`. That way it doesn't matter what order the list is in, as long as those movies are in the list somewhere.